src/components/toaster/toaster-context.tsx

```diff
@@ -0,0 +1,63 @@
+import { createContext, useContext, useState } from 'react';
```

Comment:
Should I put _toaster-context_ file aside with _cart-open-context_, in the lib folder?